folium/features.py

```diff
@@ -758,13 +758,18 @@ class MarkerCluster(Layer):
         Whether the Layer will be included in LayerControls
 
     """
-    def __init__(self, name=None, overlay=True, control=True):
+    def __init__(self, name=None, overlay=True, control=True, icon_create_function=""):
         super(MarkerCluster, self).__init__(name=name, overlay=overlay,
                                             control=control)
         self._name = 'MarkerCluster'
+        self._icon_create_function = icon_create_function.strip()
         self._template = Template(u"""
             {% macro script(this, kwargs) %}
-            var {{this.get_name()}} = L.markerClusterGroup();
+            var {{this.get_name()}} = L.markerClusterGroup({
+                {% if this._icon_create_function != "" %}
```

Comment:
I believe that `{% if this._icon_create_function %}` would work.
Also, maybe a default to `None` instead of `''`would be more readable.

Reply:
Yes, you're right. It works.